linchpin/examples/controller-provisioning/README.md

```diff
@@ -0,0 +1,34 @@
+Example: Provisioning of *controller* by kickstart installation
+---------------------------------------------------------------
+
+## Credentials configuration:
+
+### Provisioning:
+
+Kickstart to be used for controller installation: [ks.kstest-controller.cfg](ks.kstest-controller.cfg)
+
+- public ssh key to access the host is passed in `sshkey` command
+
+Inventory for the deployment playbook: [hosts](hosts)
```

Comment:
The `(hosts)` should be a path so please change this to `./hosts`.

Reply:
`hosts` is a relative path, no?
I am using relative paths to a file like this in many other links.

Reply:
OK in that case you can leave it this way. It's only a more obvious that it is a relative path with the `./` but I'm fine even with this.